README.md

```diff
@@ -11,6 +11,17 @@
 
 For documentation, visit [https://frint.js.org](https://frint.js.org).
 
+Key characteristics and features include:
+
+* Gives your applications a **structure**
+* **Environment** agnostic (runs in browser, server, and CLI)
+* **Rendering** library agnostic (integrates with React, Vue, and Preact)
+* **Composable** with multiple packages as needed
+* Each package focused on doing **one thing** well only
+* **Modular** architecture with Apps
+* Embraces **reactive programming** with RxJS
+* **Progressive** and easy to adopt it in existing applications
```

Comment:
I don't think you need "it" in this sentence. Because the sentence already implies "It is" in the beginning:
"(It is) progressive and easy to adopt in existing applications".